bin/generate_tests.py

```diff
@@ -12,6 +12,13 @@
     generate_tests.py --check           Checks if test files are out of sync with templates
     generate_tests.py --check two-fer   Checks if two-fer test file is out of sync with template
 """
+import sys
+
+_py = sys.version_info
+if _py.major < 3 or (_py.major == 3 and _py.minor < 7):
+    print("Python version must be at least 3.7")
```

Comment:
Just tested this: the generator runs fine on 3.6. 3.5 doesn't support `black==19.3b0`.

No reason we can't allow 3.6 to run the generator until we employ [walruses](https://www.python.org/dev/peps/pep-0572/) or something else new in 3.7.

Reply:
True ... though we had a soft requirement on 3.7 simply because of what was in the shebang ... do we _want_ to relax that?

Reply:
:man_shrugging: I'll be honest here: that shebang was the result of my laziness and not wanting to type out `python3.7 bin/generate_tests.py`. My logic at the time for developing the generator in 3.7 was that it was new and shiny; clearly all the features used are supported in 3.6.

Unless we can demonstrate a clear performance boost using 3.7 over 3.6, I don't have a problem relaxing the requirement.

Reply:
Righto.

Reply:
Pushed changed will allow 3.6 and up.